policy-controller/grpc/src/lib.rs

```diff
@@ -297,10 +303,24 @@ fn to_authz(
             };
 
             proto::Authz {
-                labels,
                 networks,
+                labels: labels.into(),
                 authentication: Some(authn),
             }
         }
     }
 }
+
+// === impl Labels ===
+
+impl From<Labels> for HashMap<String, String> {
+    fn from(labels: Labels) -> HashMap<String, String> {
```

Comment:
assuming this can't be a `HashMap<&'static str, String>` due to prost reasons?

Reply:
yep